### DISCUSSION TITLE
MQTT Steuerung

### DISCUSSION THREAD

**Discussion by floi001** [tbnobody/OpenDTU#1121] [Q&A]
Hi!
Ich habe gerade auf Facebook gelesen, dass man auch über MQTT das Limit steuern kann. 
Muss man dazu etwas bestimmtes einstellen? Ich bekomme alle Werte zu meine MQTT ioBroker aber wenn ich den Wert über MQTT ändere, so wird das von der OpenDTU ignoriert. 
Im Moment steuere ich über die Web API. MQTT wäre mir aber lieber.
Danke

**Comment by 3aau** [comment]
moin, da hätte ich auch interesse drann. 
wie muss die mqtt-message aussehen die an die dtu gesendet werden muss?
ich sammele die mqtt-daten auf einem kleinen teensy 4.1 ein. dieser sollte dann auch die steuerung des wr machen.

**Comment by DPO99** [comment]
Steht doch alles auf dem Github.

https://github.com/tbnobody/OpenDTU/blob/master/docs/MQTT_Topics.md

**Comment by floi001** [comment]
@DPO99 
Ja, steht in der Beschreibung - funktioniert aber über MQTT nicht.

**Reply by DPO99** [reply]
Dann erzähl doch Mal was zum Aufbau. Welcher mqtt Broker? Einstellungen des mqtt Broker etc. 


Am einfachsten kann man das über den Mqtt Explorer testen da kann man ganz easy retain und non retain einstellen.

Und welchen genauen Pfad sprichst du mit dem Test an?

Und auch gerne Mal die Diskussion hier durchsuchen da gibt es schon x Threads mit dem selben Thema. Meistens wird nicht der cmd Pfad genutzt oder es scheitert eben an den korrekten Einstellungen des Brokers.

**Reply by floi001** [reply]
Ich habe es im ioBroker laufen.
Das ganze sieht bei mir so aus:
![grafik](https://github.com/tbnobody/OpenDTU/assets/124238276/818fef3e-69f9-4817-990f-38e25fa24fad)
Egal was ich eingebe - kommt nicht an.

**Reply by DPO99** [reply]
Da gibt's auch Beiträge zu.

https://github.com/tbnobody/OpenDTU/discussions/874#discussioncomment-5783804

**Reply by floi001** [reply]
Danke!!!
